config/jobs/kubernetes/sig-k8s-infra/releng/artifact-promotion-presubmits.yaml

```diff
@@ -8,7 +8,7 @@ presubmits:
       testgrid-dashboards: sig-k8s-infra-k8sio
     decorate: true
     skip_report: false
-    run_if_changed: 'k8s.gcr.io/((images/.*/images\.yaml)|(manifests/.*/promoter-manifest\.yaml))'
+    run_if_changed: 'registry.k8s.io/((images/.*/images\.yaml)|(manifests/.*/promoter-manifest\.yaml))'
```

Comment:
this change is wrong, we're triggering some automation when the k8s.gcr.io directory in this repo changes

https://github.com/kubernetes/test-infra/pull/28674#discussion_r1102595492